weasyl/journal.py

```diff
@@ -20,7 +20,7 @@
 
 from libweasyl import text, staff
 
-from weasyl import media
+from weasyl import media, orm
```

Comment:
Is there a reason for preferring speparate lines? Seems like an unneeded extra line to me.